openedx/core/lib/xblock_utils/__init__.py

```diff
@@ -139,7 +139,7 @@ def wrap_xblock(
     template_context = {
         'content': block.display_name if display_name_only else frag.content,
         'classes': css_classes,
-        'display_name': block.display_name_with_default_escaped,
+        'display_name': block.display_name_with_default,
```

Comment:
Why this change? It doesn't look like `display_name` is used in the [xblock_wrapper.html template](https://github.com/edx/edx-platform/blob/36fc236730540c3c354933ac784249ec1b54f9cd/common/templates/xblock_wrapper.html) at all, can we just remove it?

Reply:
You're right that it's not used, which is why I was comfortable changing it, but I'm not sure why it was in here. I wondered if it might be in use in a theme or something, so I figured I'd leave it in to have less chance of breaking something.

Reply:
It's too bad we don't know where this is used (if it is used).  The point of not escaping here is because the actual escaping should be happening later.  However, if we don't know where this gets used, we can't confirm.